File: C++/Ejercicio1.cpp
#include <iostream>
using namespace std;

int main() {
    // 1.Declarar un Array en C++
    int array1[5] = {0}; // Inicializado con ceros
    
    // 2.Inicializar un Array en C++
    int array2[] = {11, 12, 13, 14, 15};
    
    // Mostrar los arrays
    cout << "Array vacío: ";
    for (int i = 0; i < 5; i++) {
        cout << array1[i] << " ";
    }
    cout << endl;
    
    cout << "Array inicializado: ";
    for (int i = 0; i < 5; i++) {
        cout << array2[i] << " ";
    }
    cout << endl;
    
    return 0;
}

File: C#/Ejercicio1.cs
using System;

// Declarar un Array en C#
int[] array1 = new int[5];

// Inicializar un Array en C#
int[] array2 = {11, 12, 13, 14, 15};

// Mostrar los arrays
Console.Write("Array vacío: ");
foreach (int num in array1) {
    Console.Write(num + " ");
}
Console.WriteLine();

Console.Write("Array inicializado: ");
foreach (int num in array2) {
    Console.Write(num + " ");
}
Console.WriteLine();